### PR TITLE
Save transfers in local store and display pending transfers in dapp

### PR DESCRIPTION
**Pull request by nephix**
Addresses #585

### CODE REVIEW

File: raiden-dapp/src/store/index.ts [hunk 1]
```diff
@@ -131,7 +141,21 @@ const store: StoreOptions<RootState> = {
...
       );
       return orderBy(openChannels, ['capacity'], ['desc'])[0];
-    }
+    },
+    pendingTransfers: ({ transfers }: RootState) =>
+      Object.keys(transfers)
+        .filter(secretHash => {
+          const { status } = transfers[secretHash];
+          const isTransferPending =
+            status !== RaidenSentTransferStatus.unlocked && // success completed case
```
Comment (andrevmatos) [inline raiden-dapp/src/store/index.ts]: Actually, pending transfers are `!transfer.completed`

Comment (nephix) [reply raiden-dapp/src/store/index.ts]: From #585 :

> Success case: PENDING -> RECEIVED -> REVEALED -> UNLOCKING -> UNLOCKED (completed, succeeded)
> Example of error case (e.g. mediators take more fees than they can): PENDING -> RECEIVED (2*reveal blocks later) EXPIRING -> EXPIRED (completed, failed)

Comment (andrevmatos) [reply raiden-dapp/src/store/index.ts]: Yes, these are information for the stepper (when/if it's implemented) of the transfer workflow. The `completed` and `success` properties of this type were created specifically for this filtering/showing, and consider some other conditions as well (e.g. transfer refunded, channel closed, etc).

Comment (nephix) [reply raiden-dapp/src/store/index.ts]: Then they should be mentioned in the ticket, not information for steppers that are not in scope at all

**Pull request merged**